src/blocktypes/devshells.nix

```diff
@@ -16,17 +17,21 @@
       flake,
       fragment,
       fragmentRelPath,
-    }: [
-      (import ./actions/build.nix flake fragment)
+      target,
+    }: let
+      developDrv = mkDevelopDrv target;
+    in [
+      (import ./actions/build.nix developDrv)
       {
         name = "enter";
         description = "enter this devshell";
         command = ''
           std_layout_dir=$PRJ_ROOT/.std
           profile_path="$std_layout_dir/${fragmentRelPath}"
           mkdir -p "$profile_path"
+          # ${developDrv}
           nix_args=(
-            "$PRJ_ROOT#${fragment}"
+            "${builtins.builtins.unsafeDiscardStringContext developDrv.drvPath}"
```

Comment:
Oversight

Reply:
the double builtins? Strangely enough it doesn't cause an error as builtins has a self recursive reference, but I can clean it up regardless.

Reply:
fixed in 25c1399ca1ae84fed1178b9aa5d6f66dca7d158a